decred/decred/dcr/vsp.py

```diff
@@ -71,15 +66,18 @@ def parse(pi):
         Args:
             pi (object): The response from the 'getpurchaseinfo' request.
         """
-        get = lambda k, default=None: pi[k] if k in pi else default
+
+        def get(k, obj, default=None):
+            return pi[k] if k in obj else default
+
```

Comment:
My editor was complaining that this was a lambda and not a definition. It seems the scopes for lambdas are different than defined functions.

Reply:
Yes, E731 from pycodestyle. It's among the ones we ignore in the Github action definition, feel free to use lambda if you think it's better.

Reply:
I'll just listen to my editor for now.